Gems/ROS2/Code/Source/RobotImporter/FixURDF/FixURDF.cpp

```diff
@@ -41,23 +43,36 @@ namespace ROS2::Utils
                 inertial = urdf->document()->allocate_node(node_element, "inertial");
                 link->append_node(inertial);
                 modified = true;
-                inertiaPresent = false;
+                inertialPresent = false;
+
+                MissingInertia missingInertia;
+                missingInertia.linkName = name;
+                missingInertias.push_back(missingInertia);
             }
 
+            IncompleteInertia incompleteInertia;
+            incompleteInertia.linkName = name;
+
             auto mass = inertial->first_node("mass");
             if (!mass)
             {
-                AZ_Warning("URDF", !inertiaPresent, "Missing mass tag inside the inertial tag link %s, applying default mass tag.", name.c_str());
+                AZ_Warning(
+                    "URDF", !inertialPresent, "Missing mass tag inside the inertial tag link %s, applying default mass tag.", name.c_str());
                 mass = urdf->document()->allocate_node(node_element, "mass");
                 mass->append_attribute(urdf->document()->allocate_attribute("value", "1."));
```

Comment:
There could be an estimation made from the mesh interior volume here (general comment, I am aware PR does not change that).